llvm/lib/Target/X86/X86ISelDAGToDAG.cpp

```diff
@@ -4087,14 +4087,17 @@ MachineSDNode *X86DAGToDAGISel::matchBEXTRFromAndImm(SDNode *Node) {
   SDValue Control;
   unsigned ROpc, MOpc;
 
+#define GET_EGPR_IF_ENABLED(OPC) Subtarget->hasEGPR() ? OPC##_EVEX : OPC
```

Comment:
(style) Add brackets just in case the define gets expanded somewhere with weird operator precedence:
`#define GET_EGPR_IF_ENABLED(OPC) (Subtarget->hasEGPR() ? OPC##_EVEX : OPC)`

Reply:
Done.